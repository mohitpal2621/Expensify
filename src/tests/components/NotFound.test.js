import React from "react";
import { render, screen } from '@testing-library/react';
import NotFound from "../../components/NotFound";
import { MemoryRouter } from "react-router-dom";


test("Should reender NOTFOUND", () => {
    const {container} = render(<MemoryRouter><NotFound/></MemoryRouter>);
    expect(container).toMatchSnapshot();
})