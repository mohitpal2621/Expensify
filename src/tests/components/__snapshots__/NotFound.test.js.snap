// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should reender NOTFOUND 1`] = `
<div>
  <div>
    404! - 
    <a
      href="/"
    >
      Go Home
    </a>
  </div>
</div>
`;
